apps/agreement/contracts/Agreement.sol

```diff
@@ -352,38 +288,55 @@ contract Agreement is IArbitrable, IForwarder, AragonApp {
             require(_canClaimSettlement(action), ERROR_CANNOT_SETTLE_ACTION);
```

Comment:
This is kind of confusing to me. The external function `canSettle` doesn’t take into account the `msg.sender`, so maybe it should include the logic in `_canClaimSettlement`. Maybe we could join these 2 internal functions in only one, which checks the sender. The fact that the error message is the same makes me think that this makes sense.